deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -834,6 +844,17 @@ def get_message(self):
                           f"- rootcheck({self.rootcheck_path})")
         return message
 
+
+class Logcollector:
+    def __init__(self):
+        self.name = 'syslog'
```

Comment:
Maybe is better to use LOGCOLLECTOR attribute name in order to follow the style of the rest of the classes of the agent simulator module.
```suggestion
        self.LOGCOLLECTOR = 'syslog'
```